app/models/spree/user_decorator.rb

```diff
@@ -18,9 +18,13 @@ def recent_order
     end
 
     def create_bronto_contact
-      contact = BrontoIntegration::Contact.new(bronto_token)
-      contact.find_or_create(email)
-      contact.update_status(email, 'active')
+      begin
+        contact = BrontoIntegration::Contact.new(bronto_token)
+        contact.find_or_create(email)
+        contact.update_status(email, 'active')
+      rescue Bronto::ValidationError
+        # noop
```

Comment:
It's also a little of my fault for not providing a block of context around the PR. Will do better next time for yah.